tensorboard/webapp/plugins/plugins.component.ts

```diff
@@ -58,10 +58,10 @@ export class PluginsComponent implements OnChanges {
   private readonly pluginInstances = new Map<string, HTMLElement>();
 
   ngOnChanges(change: SimpleChanges): void {
-    if (change.activePlugin && this.activePlugin) {
+    if (change['activePlugin'] && this.activePlugin) {
```

Comment:
This is arcane. How are we supposed to know it beforehand? Is there a way to enforce this in OSS?

Reply:
AFAIK, no :(. I kinda hope gts includes more conformance checks but there was no clear evidence that it is the case.